apps/voucher/cypress/e2e/consent-integration.cy.ts

```diff
@@ -32,22 +32,6 @@ describe("Consent integration Test", () => {
       cy.get("[data-testid=verification_code_input]").should("not.be.disabled")
       cy.get("[data-testid=verification_code_input]").type(code)
 
-      cy.contains("label", "read").should("exist")
-      cy.contains("label", "read").should("be.visible")
-      cy.contains("label", "read").should("not.be.disabled")
-      cy.contains("label", "read").should("not.be.disabled")
-      cy.contains("label", "read").click()
-
-      cy.contains("label", "write").should("exist")
-      cy.contains("label", "write").should("be.visible")
-      cy.contains("label", "write").should("not.be.disabled")
-      cy.contains("label", "write").click()
-
-      cy.get("[data-testid=submit_consent_btn]").should("exist")
-      cy.get("[data-testid=submit_consent_btn]").should("be.visible")
-      cy.get("[data-testid=submit_consent_btn]").should("not.be.disabled")
-      cy.get("[data-testid=submit_consent_btn]").click()
-
```

Comment:
same here

Reply:
It is, we're skipping consent in the tests

https://github.com/GaloyMoney/blink/pull/4352/files#diff-837b4f391cfd25241cfd9947e05f126a3078d0b3109844600ec949dcc3dd1ebdR25